test/tests/signTypedData.ts

```diff
@@ -1,4 +1,5 @@
-import { EIP712Message, Permit2 } from '../../app/ts/utils/wire-types.js'
+import { EIP712Message } from '../../app/ts/utils/wire-types.js'
+import { Permit2 } from '../../app/ts/utils/personal-message-definitions.js'
```

Comment:
How did this not fail CI previously?  Sounds like a bug in our CI process?

Reply:
It did fail the CI. I forgot to wait for CI to complete before merging (as CI is not run to branches not merging to main). I wonder if we could get it to run for those too?

Reply:
I tried to create release and then I noticed it was failing for this: <https://github.com/DarkFlorist/TheInterceptor/actions/runs/4858826351>